### PR TITLE
i18n(pt-BR): Update `cli-reference.mdx` translation

### PR DESCRIPTION
**Pull request by Egpereira**
# Update `cli-reference.mdx` translation

## Description (required)

Update the `cli-reference.mdx` with the latest changes.

### CODE REVIEW

File: src/content/docs/pt-br/reference/cli-reference.mdx [hunk 1]
```diff
@@ -243,25 +224,21 @@ Use estas flags para customizar o comportamento do comando.
...
 
-O comando vai observar qualquer mudança nos arquivos `.astro`, e vai reportar qualquer erro.
-
-:::note
-Este comando apenas checa a tipagem em arquivos `.astro`.
-:::
+O comando observará qualquer alteração em seu projeto e comunicará qualquer erro.
 
-📚 Leia mais sobre o [suporte para TypeScript no Astro](/pt-br/guides/typescript/).
+<ReadMore>Leia mais sobte [checagem de tipos no Astro](/pt-br/guides/typescript/#checagem-de-tipos).</ReadMore>
```
Comment (yanthomasdev) [inline src/content/docs/pt-br/reference/cli-reference.mdx:229]:
```suggestion
<ReadMore>Leia mais sobre [checagem de tipos no Astro](/pt-br/guides/typescript/#checagem-de-tipos).</ReadMore>
```

File: src/content/docs/pt-br/reference/cli-reference.mdx [hunk 2]
```diff
@@ -495,3 +532,17 @@ const codigoSaida = await sync({
 
 process.exit(codigoSaida)
 ```
+
+## Astro Studio CLI
```
Comment (yanthomasdev) [inline src/content/docs/pt-br/reference/cli-reference.mdx:536]:
```suggestion
## CLI do Astro Studio
```

File: src/content/docs/pt-br/reference/cli-reference.mdx [hunk 3]
```diff
@@ -495,3 +532,17 @@ const codigoSaida = await sync({
...
+
+### `astro login`
+
+Autenticar com o Astro Studio. Isso é necessário para executar todos os comandos de gerenciamento de banco de dados, incluindo [`astro link`](#astro-link).
```
Comment (yanthomasdev) [inline src/content/docs/pt-br/reference/cli-reference.mdx:540]:
```suggestion
Autentica com o Astro Studio. Isso é necessário para executar todos os comandos de gerenciamento de banco de dados, incluindo [`astro link`](#astro-link).
```

File: src/content/docs/pt-br/reference/cli-reference.mdx [hunk 4]
```diff
@@ -495,3 +532,17 @@ const codigoSaida = await sync({
...
+
+### `astro link`
+
+Link para seu banco de dados hospedado no Studio. Isso é necessário para executar [comandos do Astro DB](/pt-br/guides/integrations-guide/db/#astro-db-cli-reference) para o gerenciamento do banco de dados. Para vincular um banco de dados, você será solicitado a selecionar um banco de dados existente ou criar um novo.
```
Comment (yanthomasdev) [inline src/content/docs/pt-br/reference/cli-reference.mdx:544]: Aqui o primeiro "link" está no sentido de "vincular" 
```suggestion
Vincula seu banco de dados hospedado no Studio. Isso é necessário para executar [comandos do Astro DB](/pt-br/guides/integrations-guide/db/#astro-db-cli-reference) para o gerenciamento do banco de dados. Para vincular um banco de dados, você será solicitado a selecionar um banco de dados existente ou criar um novo.
```

File: src/content/docs/pt-br/reference/cli-reference.mdx [hunk 5]
```diff
@@ -495,3 +532,17 @@ const codigoSaida = await sync({
...
+
+### `astro logout`
+
+Faça logout do Astro Studio e remova todas as chaves de autenticação armazenadas localmente.
```
Comment (yanthomasdev) [inline src/content/docs/pt-br/reference/cli-reference.mdx:548]:
```suggestion
Faz logout do Astro Studio e remove todas as chaves de autenticação armazenadas localmente.
```

**Review by yanthomasdev — COMMENTED** [review]
Thanks @Egpereira, I've left a few suggestions 🙌

**Comment by Egpereira** [comment]
Sorry for the delay @yanthomasdev, applied your suggestions!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**